docs/swagger.yaml

```diff
@@ -291,9 +283,7 @@ paths:
         - ccx
     patch:
       operationId: ccx_v0_ccx_partial_update
-      summary: Modifies a CCX course.
-      description: "Args:\n    request (Request): Django request object.\n    ccx_course_id\
-        \ (string): URI element specifying the CCX course location."
+      description: Modifies a CCX course.
```

Comment:
Sorry, I may have approved to eagerly.

This looks like a regression in the doc tools. This is what we were talking about on Slack, right? Unless you were about to address it, can you add an issue to https://github.com/edx/api-doc-tools/issues ?

Reply:
I need to look into it more. Right now it seems like:
- Only the paragraph parts of the docstring are kept for swagger.yaml.  Args/Returns/etc don't seem to be kept. That makes sense. Those are Python constructs, not REST.
- If there is one paragraph in the docstring, it becomes the description.
- If there is more than one paragraph, the first becomes the summary, and the rest become the description.